LICENSE-MIT

```diff
@@ -0,0 +1,16 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy of
```

Comment:
According to this: https://opensource.org/licenses/MIT isn't this license missing `Copyright <YEAR> <COPYRIGHT HOLDER>` at the top?

Reply:
I don't believe its required - rust doesn't have it on theirs.